internal/searcher/model.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 Jetpack Technologies Inc and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+
+package searcher
+
+type NixPackageInfoList []*NixPackageInfo
+
+type NixPackageInfo struct {
+	AttributePath string  `json:"attribute_path,omitempty"`
+	NixpkgCommit  string  `json:"nixpkg_commit,omitempty"`
+	PName         string  `json:"pname,omitempty"`
+	Version       string  `json:"version,omitempty"`
+	Score         float64 `json:"score,omitempty"`
+}
+
+type Result struct {
+	Name     string             `json:"name"`
```

Comment:
Is this the pname or the attribute path? seems like the attribute path right now.

Reply:
Attribute path.